### PR TITLE
Skip test on okd

### PR DESCRIPTION
**Pull request by SchSeba**
**What this PR does / why we need it**:
This PR add a skip for the test:

should create a virtual machine with one interface with network definition from different namespace
as it's not supported on okd

**Special notes for your reviewer**:
waiting for https://github.com/kubevirt/kubevirt/pull/2623

**Release note**:

```release-note
NONE
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt/2700)

### CODE REVIEW

**Comment by SchSeba** [comment]
/hold

Put a "F" to validate the skip

File: tests/tests_suite_test.go [hunk 1]
```diff
@@ -32,7 +32,11 @@ import (
 )
 
 func TestTests(t *testing.T) {
-	reporters := []Reporter{reporter.NewKubernetesReporter(os.Getenv("ARTIFACTS"))}
+	artifactsDir := os.Getenv("ARTIFACTS")
+	if artifactsDir == "" {
```
Comment (fedepaol) [inline tests/tests_suite_test.go]: The default is already provided by hack/functests.sh
https://github.com/kubevirt/kubevirt/blob/master/hack/functests.sh#L24

By doing this, there won't be a way for not having the reporter logs dumped.

**Comment by SchSeba** [comment]
close this in favor of https://github.com/kubevirt/kubevirt/pull/2623

**Pull request closed without merging**